src/acom_music_box/tools/waccmToMusicBox.py

```diff
@@ -103,7 +188,7 @@ def readWACCM(waccmMusicaDict, latitude, longitude,
 
     # open dataset for reading
     waccmDataSet = xarray.open_dataset("{}/{}".format(modelDir, waccmFilename))
-    if (True):
+    if (False):
         # diagnostic to look at dataset structure
         logger.info("WACCM dataset = {}".format(waccmDataSet))
```

Comment:
is this still needed, or was it just for debugging?

Reply:
No longer needed (just for debugging).